src/api/NotificationsAssistant.ts

```diff
@@ -98,22 +102,33 @@ export default class NotificationsAssistant{
 	async ClearNotifications(){
 
 		const pending = await Plugins.LocalNotifications.getPending()
-		Plugins.LocalNotifications.cancel(pending) //clear all the pending notifications
+		if(pending.notifications.length != 0){
+			Plugins.LocalNotifications.cancel(pending)
+		}
+		 //clear all the pending notifications
 		this.frequency = 0
 		this.start = 0
 		this.end = 0
 		this.verses = []		
 	}
 
+	async NoNotificationsRemaining() : Promise<Boolean>{
```

Comment:
return type should be of Promise<boolean> rather than Promise<Boolean>. Use primitive types when possible